EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -899,7 +899,7 @@ void V2::World::allocateFactories(const EU4::World& sourceWorld)
 		admMean += (admTech - admMean) / num;
 		++num;
 	}
-
+	
```

Comment:
Spurious tab.

Reply:
I know! I have no clue how to prevent those. I don't even see them in VS until I push to Github and VS seems to generate them automatically. Help! :)